conformance/1.0/kfp-conformance.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kfp-conformance
+  namespace: kf-conformance
```

Comment:
Is this namespace already created in Kubeflow cluster? Should we create this namespace first?

Reply:
The namespace is created by the profile in setup.yaml

Reply:
Sounds good!